AFTests/AlgoStore/AlgoStoreTests.cs

```diff
@@ -808,5 +809,36 @@ public partial class AlgoStoreTests : AlgoStoreTestDataFixture
             // Stop the algo instance
             await AlgoStoreCommonSteps.StopAlgoInstance(Consumer, postInstanceData);
         }
+
+        [Test, Description("AL-488")]
+        [Category("AlgoStore")]
+        public async Task CheckTokenGeneratedOnInstanceCreation()
+        {
+            Dictionary<string, string> queryParams = new Dictionary<string, string>()
+            {
+                { "algoId" , postInstanceData.AlgoId },
+                { "instanceId", postInstanceData.InstanceId}
+            };
+
+            var instanceDataResponse = await Consumer.ExecuteRequest(algoInstanceDataPath, queryParams, null, Method.GET);
+            Assert.That(instanceDataResponse.Status, Is.EqualTo(HttpStatusCode.OK));
+
+            // Wait up to 3 minutes for the algo to be started
+            await AlgoStoreCommonSteps.WaitAlgoToStart(ClientInstanceRepository, postInstanceData);
+
+            // Get Algo Instance Data from DB
+            ClientInstanceEntity instanceDataFromDB = await ClientInstanceRepository.TryGetAsync(t => t.Id == postInstanceData.InstanceId) as ClientInstanceEntity;
+
+            // Assert AuthToken is not null
+            Assert.That(instanceDataFromDB.AuthToken, Is.Not.EqualTo(null));
```

Comment:
It can be simplified : Assert.That(instanceDataFromDB.AuthToken, Is.Not.Null, "some message if Null");